spec/graphql/mutations/customers/update_spec.rb

```diff
@@ -21,6 +21,7 @@
           canEditAttributes
           invoiceGracePeriod
           providerCustomer { id, providerCustomerId }
+          billingConfiguration { documentLocale }
```

Comment:
Maybe request the ID to make sure it's present 🙏

Reply:
added